modules/snooker/src/main/ts/ephox/snooker/operate/Render.ts

```diff
@@ -4,6 +4,7 @@ import { Attribute, Css, Insert, InsertAll, SugarElement } from '@ephox/sugar';
 export interface RenderOptions {
   styles: Record<string, string>;
   attributes: Record<string, string>;
+  useColumnGroups: boolean;
```

Comment:
The `use` here doesn't really make a lot of sense and doesn't add any value, so I'd suggest just calling this `colGroups`. That way it's shorter (remember object properties often won't be minified), matches the actual element name and is still readable.